vello_tests/tests/smoke_snapshots.rs

```diff
@@ -112,3 +112,31 @@ fn two_emoji_gpu() {
 fn two_emoji_cpu() {
     two_emoji(true);
 }
+
+#[test]
+#[cfg_attr(skip_gpu_tests, ignore)]
+fn test_bgra() {
+    let mut scene = Scene::new();
+    let mut blob: Vec<u8> = Vec::new();
+    [
+        palette::css::RED,
+        palette::css::LIME,
+        palette::css::BLUE,
+        palette::css::WHITE,
+    ]
+    .iter()
+    .for_each(|c| {
+        blob.extend(c.to_rgba8().to_u8_array());
```

Comment:
Is there a reason we have not swapped the `r` and `b` here?

We seem to currently be testing the ability to render "actually rgba" images wrong, rather than testing the ability to render bgra images correctly.